### ISSUE TITLE
Issue with making hidden field in create and edit actions, which changes positions of other form fields.

### ISSUE THREAD

**Issue by SuperMasterBlasterLaser** [sferik/rails_admin#2487]
Hello.

I have updated rails_admin to 0.8.\* version.

In my `create` and `edit` actions, there is hidden field which will assign id of current user that logged in:

```
   edit do
      include_all_fields
      field :updater_id, :hidden do
        visible true
        def value
           bindings[:view]._current_user.id
        end
     end
   end
```

When I open my page it shows me this:
![1](https://cloud.githubusercontent.com/assets/8684688/11360204/8620fa7e-92ac-11e5-8ea1-869249c5cb4b.PNG)

Then I opened html code of this:

![2](https://cloud.githubusercontent.com/assets/8684688/11360213/9f33dfcc-92ac-11e5-8675-f2d6e3fb0a1e.PNG)

According to source of this page. It creates `col-sm-10` column for hidden field which moves next field.

**Comment by mshibuya** [comment]
Merged #3278.

**Comment by SuperMasterBlasterLaser** [comment]
@mshibuya  Finally after five years. 😂🎈✨🎉🎉
